tests/TestGraph.hs

```diff
@@ -20,13 +17,12 @@ spec = do
     let modulePaths = (sourcesDir <>) <$> ["Module.purs", "Module2.purs", "Module3.purs"]
     let graphFixtureName = "graph.json"
 
-    graphFixture <- readUTF8FileT (baseDir <> graphFixtureName)
+    Just graphFixture <- Json.decodeFileStrict' (baseDir <> graphFixtureName)
     eitherGraph <- fst <$> P.graph modulePaths
     case eitherGraph of
       Left err -> error $ "Graph creation failed. Errors: " <> show err
       Right res ->
-        let textRes = Text.decodeUtf8 $ ByteString.toStrict $ Json.encode res
-        in graphFixture `shouldBe` textRes
+        res `shouldBe` graphFixture
```

Comment:
This change means that we don't have to update the fixture if the order of fields changes, and we don't get errors if the fields end up in different orders due to different build configurations. Updating aeson seems to have caused the fields in the result to end up in a different order.

Reply:
Also, you're supposed to put the expected value second. The error message describes the 'expected' vs 'actual' value, so the error can be misleading if you put the expected value first; this is why I've flipped them around.